package.json

```diff
@@ -2,6 +2,10 @@
   "version": "1.6.3",
```

Comment:
@wrux Shouldn't we commit the package-lock.json file when removing the yarn.lock file?

Reply:
Both existed in the repo and I decided to remove yarn seeing as we use package-lock in basically all projects.